### ISSUE TITLE
Mash issue in upgrade

### ISSUE THREAD

**Issue by raw937** [gbouras13/pharokka#315]
* pharokka version: 1.5.1 (stated 1.4 databases)
* Python version: 3.7
* Operating System: Linux

### Description

Traceback (most recent call last):
  File "/home/docwhite/miniconda3/envs/pharokka/bin/pharokka.py", line 496, in <module>
    main()
  File "/home/docwhite/miniconda3/envs/pharokka/bin/pharokka.py", line 138, in main
    ) = check_dependencies(args.skip_mash)
  File "/home/docwhite/miniconda3/envs/pharokka/bin/input_commands.py", line 528, in check_dependencies
    mash_major_version = int(mash_out.split(".")[0])
ValueError: invalid literal for int() with base 10: 'mash: error while loading shared libraries: libgsl'

Some type of mash error related to string to int in python. Not a valid int. 

### What I Did

```
conda create -n pharokka
conda activate pharokka
mamba install -c bioconda pharokka
install_databases.py -o ~/miniconda3/envs/pharokka/databases
pharokka.py -i CB_2.fna -o CB2 -d ~/miniconda3/envs/pharokka/databases/ -t 8 --fast -g prodigal-gv 
pharokka.py -i CB_2.fna -o CB2 -d ~/miniconda3/envs/pharokka/databases/ -t 8 --fast -g prodigal
```

**Comment by gbouras13** [comment]
Hi @raw937,

This issue is similar to #235 and #240 . I need to add better exception handling (on the list for v1.6!) but essentially the error means mash didn't install properly via conda.

Try installing v2.2

```
mamba install -c bioconda mash=2.2
```

George

**Comment by raw937** [comment]
That fixed it. Running now.

**Comment by camilogarciabotero** [comment]
Hi George,

I did have this issue in the latest (1.6) pharokka version. I was installing it through conda/mamba. And it was solved after `mamba install -c bioconda mash=2.2`. 

Best
